assets/helpers/render.js

```diff
@@ -3,11 +3,11 @@
 import ReactDOM from 'react-dom';
 import { logException } from 'helpers/logger';
 
-const renderPage = (content: Object, id: string) => {
+const renderPage = (content: Object, id: string, callBack?: () => void) => {
```

Comment:
We have to do this because before we render we set some state variables in `ContributionsLandingInit`, and this was invalidating the form because it was trying to access the DOM elements which didn't exist yet. We can remove this once we stop using the constraints validation API all together, which we have planned.

Reply:
Nice of `ReactDom.render` to accept a callback argument!